articles/key-design-of-hdfs-architecture/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: engineering-education
+status: 
+published: 
+url: /engineering-education/key-design-of-hdfs-architecture/
+title: Key Design of HDFS architecture
+description: This article will describe a brief introduction to Big data Framework, the general Design of HDFS architecture, ways of configuring HDFS Data Storage policies, Colocation and it's benefits in HDFS.
+author: ruth-mare
+date: 2021-03-19T00:00:00-14:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/key-design-of-hdfs-architecture/hero.jpg
+    alt: HDFS Architecture cover image 
+---
+HDFS (Hadoop Distributed File System) is a big data distributed file system storage by Apache that is implemented within the hadoop framework and that needs to have several features of design implemented to work effectively in processing, distibuting and storing big data.
+
+<!--more-->
+
+### Overview
+This article will cover:
+- [ntroduction to Big Data Framework](#Introduction-to-Big-Data-framework)
+- [General Design of HDFS architecture](#General-Design-of-HDFS-architecture)
+- [Configuring HDFS data storage policies](#Configuring-HDFS-data-storage-policies)
+- [Colocation and its benefits in HDFS](#Colocation-and-its-benefits-in-HDFS)
```

Comment:
On the markdown headers part, you must write everything is lower case. Something like this:

`- [Introduction to Big Data Framework](#introduction-to-big-data-framework)`

Make the changes for the remaining headings.